packages/labs/cli/package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "name": "@lit-labs/cli",
   "description": "Tooling for Lit development",
-  "version": "0.0.0",
+  "version": "0.0.1",
```

Comment:
Should not bump. Why didn't private work??

Reply:
I think it's because there's already a temp changeset md file present. I'll remove that so it doesn't get bumped. I wonder if going forward the changset action will see the private flag and not complain that it's missing a changeset?

Reply:
sgtm